src/core/plugins/postmarketos/plugin.spec.js

```diff
@@ -78,12 +77,12 @@ describe("postmarketos plugin", () => {
 
       await pmosPlugin.action__rename_unpacked_files({ group, files });
       expect(pmosPlugin.event.emit).toHaveBeenCalledTimes(3);
-      expect(fs.renameSync).toHaveBeenCalledWith(
-        path.join(basepath, "somethingelse.img.xz"),
+      expect(fs.rename).toHaveBeenCalledWith(
+        path.join(basepath, "somethingelse.img"),
```

Comment:
This is how the function works, it doesn't operate on `.img.xz` files, `.xz` is [stripped with `.replace`](https://github.com/calebccff/ubports-installer/pull/2/files#diff-8de4bb1a375c7ee2f9862dc10f08809158f384f13b2d6ba78f7bd659134b604bR91) before `fs.rename` is called.